packages/blocks/src/_common/inline/presets/nodes/reference-node/reference-popup.ts

```diff
@@ -283,7 +283,7 @@ export class ReferencePopup extends WithDisposable(LitElement) {
               <icon-button
                 size="24px"
                 class="affine-reference-popover-view-selector-button embed card-view"
-                .hover=${false}
+                hover=${false}
```

Comment:
Not a lit expert, what's the difference between `hover`, `.hover` and `?hover` ?

Reply:
In Lit, you can bind properties, attributes, and boolean attributes in templates using specific syntax. The differences between `.hover=${}`, `?hover=${}`, and `hover=${}` are related to how these bindings are applied to the elements in your template.

### `hover=${}`

This is a standard attribute binding. It sets the attribute on the element to the given value.

#### Example
```javascript
import { LitElement, html, css, customElement } from 'lit';

@customElement('my-element')
class MyElement extends LitElement {
  render() {
    const hoverValue = 'hovered';
    return html`<div hover=${hoverValue}>Content</div>`;
  }
}
```

In this example:
- The `hover` attribute is set to the value of `hoverValue` (`'hovered'`).
- This results in `<div hover="hovered">Content</div>` being rendered.

### `.hover=${}`

This syntax binds a property of the element to the given value, rather than setting an attribute. This is used when you want to directly set a property on the DOM element.

#### Example
```javascript
import { LitElement, html, css, customElement } from 'lit';

@customElement('my-element')
class MyElement extends LitElement {
  render() {
    const hoverValue = true;
    return html`<div .hover=${hoverValue}>Content</div>`;
  }
}
```

In this example:
- The `hover` property of the `div` element is set to `true`.
- This does not reflect in the HTML as an attribute but sets the DOM property directly.

### `?hover=${}`

This syntax is used for boolean attributes. If the value is true, the attribute is added; if false, the attribute is removed.

#### Example
```javascript
import { LitElement, html, css, customElement } from 'lit';

@customElement('my-element')
class MyElement extends LitElement {
  render() {
    const isHovered = true;
    return html`<div ?hover=${isHovered}>Content</div>`;
  }
}
```

In this example:
- If `isHovered` is `true`, the `hover` attribute is present on the `div` element (`<div hover>Content</div>`).
- If `isHovered` is `false`, the `hover` attribute is not present on the `div` element.

### Summary

- **`hover=${}`**: Standard attribute binding. Sets the attribute to the given value as a string.
- **`.hover=${}`**: Property binding. Sets the property on the element to the given value.
- **`?hover=${}`**: Boolean attribute binding. Adds or removes the attribute based on the boolean value.

Each of these bindings serves different purposes and allows you to interact with elements in specific ways. Understanding these differences helps you effectively manage how attributes and properties are applied in your Lit templates.

Reply:
I'm not a lit expert either, this is what I learned from chatgpt ;)